embedded/tbtree/tbtree.go

```diff
@@ -1907,17 +1922,22 @@ func (n *innerNode) split() (node, error) {
 		nodes: n.nodes[splitIndex:],
 		mut:   true,
 	}
-
 	newNode.updateTs()
 
 	n.nodes = n.nodes[:splitIndex]
-
 	n.updateTs()
 
-	metricsBtreeInnerNodeEntries.WithLabelValues(n.t.path).Observe(float64(len(n.nodes)))
-	metricsBtreeInnerNodeEntries.WithLabelValues(n.t.path).Observe(float64(len(newNode.nodes)))
```

Comment:
Some metrics were removed here

Reply:
split is a recursive function and the base case should cover this metrics... could you double check it?

Reply:
Yup, you're right, if we do recursive split for new two nodes, node sizes will be observed